core/tasks.py

```diff
@@ -220,6 +219,9 @@ def deferred(self, *args, **kwargs):
                                             retryCount))
         cmd, data = json.loads(req.body, object_hook=jsonDecodeObjectHook)
         funcPath, args, kwargs, env = data
+        if conf.get("viur.tasks.logs", False):
```

Comment:
Instead of wrapping a logging statement with an if, you should better use a logger instance which isn't the root-logger and is selectively controllable.


Create a new logger at the begin of the file (after the imports):
```py
logger = logging.getLogger(__name__)
```
and set the level semi-high:
```py
logger.setLevel(logging.WARNING)
```

If you need the debugging output, you can decrease the level in your project main.py 
```py
logging.getLogger("viur.core.tasks").setLevel(logging.WARNING)
```

---

IMO this should be `debug` and not `info` level. With the changes above and in a more short version it could be:
```py
logger.debug(f"Call task {funcPath} with {cmd=} {args=} {kwargs=} {env=}")
```

Reply:
@sveneberth this is a cool feature with the `{cmd=}` in f-strings, didn't know about it, yet!